src/com/page/event/mygrades/mygrades.js

```diff
@@ -94,10 +86,10 @@ export default class MyGrades extends Component {
 		promises.push($Node.Get(eventId).then(r => {
 			const {meta} = r.node[0];
 			let gradeIndex = 1;
-			while (true) {
+			while ( true ) {
 				const gradeKey = `grade-${pad(gradeIndex, 2)}`;
```

Comment:
Backtick strings should not be used.

Reply:
Why is that btw? It makes it so so much more readable.